src/frontend/src/optimizer/property/monotonicity.rs

```diff
@@ -43,7 +46,7 @@ impl MonotonicityDerivation {
 }
 
 /// Represents the monotonicity of a column. `NULL`s are considered largest when analyzing monotonicity.
```

Comment:
It might worth documenting what's meaning of monotonicity for distributed operators - monotonic in each shard?

Reply:
And what's the meaning for updatable stream - ignore op? This might make the concept a little strange.

Reply:
Comments added.